### PR TITLE
practice of writing a metric

### PR DESCRIPTION
**Pull request by robi111432**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,3 +1,4 @@
+This is just a test line. will be removed shortly
```
Comment (scopatz) [inline README.rst]: Please remove this line

File: cymetric/metrics.py [hunk 2]
```diff
@@ -410,6 +410,26 @@ def annual_electricity_generated_by_agent(series):
...
+
+# Mass of SNF+HLW disposed
+dependencies = [
+	('Mass', (Col1), 'Value1')
+	('Energy', (Col2), 'Value2')
+	]
+
+schema = [('SimId', ts.INT), ('Mass', ts.INT), ('Energy', ts.DOUBLE)]
+
+@metric(name='MassOfSNF+HLWdisposedPerEnergyGenerated', depends=dependencies, schema=schema)
```
Comment (scopatz) [inline cymetric/metrics.py]: Metric names shouldn't have special characters. Please replace the `+` with something else.

File: cymetric/metrics.py [hunk 3]
```diff
@@ -410,6 +410,26 @@ def annual_electricity_generated_by_agent(series):
...
+def mass_of_SNF_HLW_disposed_per_energy_generated(series):
+	"""Mass of SNF+HLW per energy generated metric returns the total mass of Spent Nuclear Fuel and HLW per the unit energy generated,
+	 calculated by total mass of SNF+HLW generated per year divided by the total energy generated per year"""
```
Comment (scopatz) [inline cymetric/metrics.py]: PEP8: lines can be at most 80 characters long. These are too lone.

File: cymetric/metrics.py [hunk 4]
```diff
@@ -410,6 +410,26 @@ def annual_electricity_generated_by_agent(series):
...
+	one = series[0]
+	two = series[1]
+	return dataframe
```
Comment (scopatz) [inline cymetric/metrics.py]: You need to actually create the dataframe in this function.

**Comment by scopatz** [comment]
Thanks for putting this in @robi111432!

File: README.rst (hunk 1 above)
Comment (robi111432) [reply README.rst]: removed already. Do I need to pull up another request after removing it?

**Comment by robi111432** [comment]
@scopatz @FlanFlanagan

File: cymetric/metrics.py [hunk 5]
```diff
@@ -410,13 +410,59 @@ def annual_electricity_generated_by_agent(series):
...
+    calculated by total mass of SNF+HLW generated per year divided by
+    the total energy generated per year"""
+    mass = series[1].reset_index()
+    mass = pd.DataFrame(data={'SimId': mass.SimId,
+                              'AgentId': mass.AgentId,
+                              'Time': mass.Time,
+                              'Mass': mass.Value},
+                        columns=['SimId', 'AgentId', 'Time', 'Mass'])
+    mass_index = ['SimId', 'AgentId', 'Time']
+    #mass = mass.groupby(mass_index).sum()
```
Comment (scopatz) [inline cymetric/metrics.py]: Please remove commented lines.

File: cymetric/metrics.py [hunk 6]
```diff
@@ -410,13 +410,59 @@ def annual_electricity_generated_by_agent(series):
...
+
+
```
Comment (scopatz) [inline cymetric/metrics.py]: PEP8: only one blank line is allowed

File: cymetric/metrics.py [hunk 7]
```diff
@@ -410,13 +410,59 @@ def annual_electricity_generated_by_agent(series):
...
+    elec = series[0].reset_index()
+    elec = pd.DataFrame(data={'SimId': elec.SimId,
+                              'AgentId': elec.AgentId,
+                              'Time': elec.Time,
+                              'Energy': elec.Value},
+                        columns=['SimId', 'AgentId', 'Time', 'Energy'])
+    el_index = ['SimId', 'AgentId', 'Time']
+    #elec = elec.groupby(el_index).sum()
```
Comment (scopatz) [inline cymetric/metrics.py]: Please remove commented line

**Comment by scopatz** [comment]
Hi @robi111432 - great start on this.  This PR also needs to have a test.  You can see the existing tests in the `tests/` directory. You probably what to add the test to the `tests/test_metrics.py`.  Also, you can run the tests using the command `nosetests`.  You can install this command with `conda install nose`

**Comment by bam241** [comment]
this hasn't been touch in 2y.

Closing for inactivity. Fell free to reopen if required.

**Pull request closed without merging**